trailblazer/dto/create_analysis_request.py

```diff
@@ -8,6 +8,7 @@ class CreateAnalysisRequest(BaseModel):
     email: str | None = None
     config_path: str
     out_dir: str
+    order_id: int | None = None
```

Comment:
Shouldn't an order_id always exist, analysis are run after an order is placed after all? Also continueing this thought - should we set order_id in the `Analysis` model with nullable= false?

Reply:
Yeah, that is the goal, although since no analyses currently have an order_id, and no order_id is sent in the request to Trailblazer, we will fix that later on. If it was set as not nullable now, no analyses could be added until a patch in cg was done.